Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/LogExceptionAnalyzer.cs

```diff
@@ -112,10 +112,18 @@ public void ReportParsingError(CompilationAnalysisContext context)
 			private bool IsCallingLogMethod(SyntaxNodeAnalysisContext context, InvocationExpressionSyntax invocation)
 			{
 				var isLoggingMethod = false;
-				if (invocation.Expression is MemberAccessExpressionSyntax memberAccess &&
-					context.SemanticModel.GetSymbolInfo(memberAccess.Expression).Symbol is INamedTypeSymbol typeSymbol)
+				if (invocation.Expression is MemberAccessExpressionSyntax memberAccess)
 				{
-					isLoggingMethod = typeSymbol.GetMembers(memberAccess.Name.Identifier.Text).OfType<IMethodSymbol>().Any(Helper.ForAllowedSymbols.IsAllowed);
+					// Get the symbol for the member access expression 
+					ISymbol memberSymbol = context.SemanticModel.GetSymbolInfo(memberAccess.Expression).Symbol;
+
+					// Use TypesHelper to get the actual type symbol
+					INamedTypeSymbol typeSymbol = new TypesHelper().GetTypeSymbol(memberSymbol);
```

Comment:
Dogfood pipeline failed here: Do not inline the constructor call for class TypesHelper